apache2/msc_util.c

```diff
@@ -105,225 +105,101 @@ int swap_int32(int x) {
  */
 char *utf8_unicode_inplace_ex(apr_pool_t *mp, unsigned char *input, long int input_len, int *changed) {
     int unicode_len = 0, length = 0;
-    unsigned int d = 0, count = 0;
+    unsigned int d = 0;
     unsigned char c, *utf;
     char *rval, *data;
     unsigned int i, len, j;
     unsigned int bytes_left = input_len;
     unsigned char *unicode = NULL;
 
+    if (input == NULL) return NULL;
```

Comment:
Is `NULL` allowed here? If not, we could use `assert()`.

Reply:
I changed it